src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDeDistribucion.kt

```diff
@@ -0,0 +1,22 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDeDistribucion(val ciudad: Ciudad) {
+    val vendedores = mutableListOf<Vendedor>()
+
+    fun agregarVendedor(vendedor : Vendedor) {
+        if(!vendedores.contains(vendedor)) {
+            vendedores.add(vendedor)
+        }
+        else {
+            Error()
+        }
+    }
+
+    fun vendedorEstrella() = vendedores.maxBy{ v -> v.puntajeCertificaciones() }
+
+    fun puedeCubrir(ciudad : Ciudad) = vendedores.any{ v -> v.puedeTrabajarEn(ciudad) }
+
+    fun vendedoresGenéricos() = vendedores.filter { v -> v.esGenerico() }
+
+    fun esRobusto() = vendedores.filter { v -> v.esFirme() }.size >= 3
```

Comment:
Bien, aunque otra forma de hacerlo es usando `count`:

```suggestion
    fun esRobusto() = vendedores.count { v -> v.esFirme() } >= 3
```